dll/win32/shdocvw/CMakeLists.txt

```diff
@@ -19,12 +17,23 @@ add_library(shdocvw MODULE
     ${SOURCE}
     mrulist.cpp
     shdocvw.rc
+    $<TARGET_OBJECTS:shdocvw_sublib>
     ${CMAKE_CURRENT_BINARY_DIR}/shdocvw.def)
 
-set_module_type(shdocvw win32dll)
+target_include_directories(shdocvw BEFORE PRIVATE
+    ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine
+    ${REACTOS_SOURCE_DIR}/sdk/lib/atl)
```

Comment:
We should use [the `atl_classes` library](https://github.com/reactos/reactos/blob/master/sdk/lib/atl/CMakeLists.txt) instead.

Reply:
```suggestion
)
```